### PR TITLE
SW-3084 Change warning for to low speeds in material settings

### PR DESCRIPTION
**Pull request by khaledsherkawi**
[SW-3084 Remove mentions of specific speed values]
 
[SW-3084 Add safety note for Finn Cardboard]
 
[SW-3084 Fix typo]
 
[SW-3084 Make safety note more prominent]
 
[SW-3084 Add speed warning upon lowering feedrate]
 
[SW-3084 Limit minimum speed based on laserhead model]
 
[SW-3084 Translate added and modified strings]

### CODE REVIEW

File: octoprint_mrbeam/static/js/app/view-models/convert.js [hunk 1]
```diff
@@ -37,7 +37,12 @@ $(function () {
                 lineDistance: 1.0,
             },
             min: {
-                speed: 30,
+                speed:
+                    MRBEAM_LASER_HEAD_MODEL === "x"
+                        ? 100
+                        : MRBEAM_LASER_HEAD_MODEL === "S"
+                        ? 50
+                        : 30,
```
Comment (JFSolution) [inline octoprint_mrbeam/static/js/app/view-models/convert.js]: Do You think we can have this in the laserhead profile? Or at least define it as constants at the beginning of the document

**Review by JFSolution — CHANGES_REQUESTED** [review]

**Comment by JFSolution** [comment]
There is also a codesmell

File: octoprint_mrbeam/static/js/app/view-models/convert.js (hunk 1 above)
Comment (khaledsherkawi) [reply octoprint_mrbeam/static/js/app/view-models/convert.js]: Done

File: octoprint_mrbeam/static/js/app/view-models/convert.js [hunk 2]
```diff
@@ -10,6 +10,10 @@ $(function () {
         self.BRIGHTNESS_VALUE_GREEN = 0.587;
         self.BRIGHTNESS_VALUE_BLUE = 0.114;
 
+        self.MINIMUM_SPEED_FOR_X_LASERHEAD = 100;
+        self.MINIMUM_SPEED_FOR_S_LASERHEAD = 50;
+        self.MINIMUM_SPEED_FOR_OHTER_LASERHEAD = 30;
```
Comment (JFSolution) [inline octoprint_mrbeam/static/js/app/view-models/convert.js]: please remove, as it is unused

**Review by JFSolution — CHANGES_REQUESTED** [review]
I would rename it to womthing like material setting min speed. As you still can go lower in the settings

File: octoprint_mrbeam/static/js/app/view-models/convert.js (hunk 2 above)
Comment (JFSolution) [inline octoprint_mrbeam/static/js/app/view-models/convert.js]:
```suggestion
```

**Review by JFSolution — DISMISSED** [review]

**Review by JFSolution — APPROVED** [review]

**Pull request merged**